### ISSUE TITLE
Pretrained model 

### ISSUE THREAD

**Issue by jamessmith90** [SeungyounShin/CAGFace#3]
Add pretrained model to measure accuracy

**Comment by jamessmith90** [comment]
Any update ?

**Comment by jiantiger** [comment]
any pre-trained model that can be donwloaded?

**Comment by HaithemH** [comment]
Any pre-trained models to provide ?
